named.go

```diff
@@ -151,7 +151,7 @@ func bindArgs(names []string, arg interface{}, m *reflectx.Mapper) ([]interface{
 	fields := m.TraversalsByName(v.Type(), names)
 	for i, t := range fields {
 		if len(t) == 0 {
-			return arglist, fmt.Errorf("could not find name %s in %#v", names[i], arg)
+			return arglist, fmt.Errorf("could not find the field named \"%s\" in \"%#v\"", names[i], arg)
```

Comment:
Sorry, you're right. You probably don't need to quote the args list as %#v
is sufficiently obvious (being a syntax representation).

On Thu, Aug 20, 2015 at 8:50 AM Carlos Alexandro Becker <
notifications@github.com> wrote:

> In named.go
> <https://github.com/jmoiron/sqlx/pull/161#discussion_r37486234>:
>
> > @@ -151,7 +151,7 @@ func bindArgs(names []string, arg interface{}, m *reflectx.Mapper) ([]interface{
> >  	fields := m.TraversalsByName(v.Type(), names)
> >  	for i, t := range fields {
> >  		if len(t) == 0 {
> > -			return arglist, fmt.Errorf("could not find name %s in %#v", names[i], arg)
> > +			return arglist, fmt.Errorf("could not find the field named \"%s\" in \"%#v\"", names[i], arg)
>
> package main
> import "fmt"
> func main() {
>     fmt.Printf("blah %q", "asdsads")
> }
>
> $ go run main.go
> blah "asdsads"
>
> I think it's double quotes...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jmoiron/sqlx/pull/161/files#r37486234>.
>